### PR TITLE
I24 serial: Beamline test fixes

### PR DESCRIPTION
**Pull request by noemifrisina**
Temporarily re-added the devices to the shell commands in the edm scripts because of [this blueapi issues](https://github.com/DiamondLightSource/blueapi/issues/647). To be removed once fixed and inject works again

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -25,7 +25,7 @@ dependencies = [
     "matplotlib",
     "nexgen",
     "numpy",
-    "opencv-python-headless",
+    "opencv-python",
```
Comment (DominicOram) [inline pyproject.toml:28]: Nit: We probably do want headless in the end, once we have removed the OAV click to move. I would add a comment to this affect

File: src/mx_bluesky/beamlines/i24/serial/extruder/EX-gui-edm/DetStage.edl [hunk 2]
```diff
@@ -113,7 +113,7 @@ font "arial-medium-r-18.0"
 buttonLabel "Move Stage"
 numCmds 1
 command {
-  0 "blueapi -c CONFIG_LOCATION controller run setup_detector_stage '\{\"expt_type\":\"Serial Jet\"\}'"
+  0 "blueapi -c CONFIG_LOCATION controller run setup_detector_stage '\{\"expt_type\":\"Serial Jet\",\"detector_stage\":\"detector_motion\"\}'"
```
Comment (DominicOram) [inline src/mx_bluesky/beamlines/i24/serial/extruder/EX-gui-edm/DetStage.edl:116]: Nit: We should try and track removing these. Either put it in the blueapi issue or make a new issue for it

**Review by DominicOram — APPROVED** [review]
Great, thank you! Couple of small optional things, take them or leave them

File: src/mx_bluesky/beamlines/i24/serial/extruder/EX-gui-edm/DetStage.edl (hunk 2 above)
Comment (noemifrisina) [reply src/mx_bluesky/beamlines/i24/serial/extruder/EX-gui-edm/DetStage.edl:116]: Made an issue

**Pull request merged**